### PR TITLE
Add "sdb" to package dependencies

### PR DESCRIPTION
**Pull request by prakashsurya**

### CODE REVIEW

**Comment by prakashsurya** [comment]
`git-ab-pre-push` is [here](http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/appliance-build-orchestrator-pre-push/2072/)

File: debian/rules [hunk 1]
```diff
@@ -53,6 +53,7 @@ DEPENDS += ansible, \
 	   openssl, \
 	   policykit-1, \
 	   rng-tools, \
+	   sdb, \
```
Comment (sdimitro) [inline debian/rules]: not a big deal but should this be grouped in the same statement with the above packages?
You don't really need `sdb` to `*create a working system*`.

Comment (prakashsurya) [reply debian/rules]: I can move it into the "for human convenience" group of packages (below), but we're going to use this in the product, so I didn't think that was appropriate.. but there's already packages in that list, that we use in the product (e.g. bcc), so maybe that's a better place to put this?

Comment (sdimitro) [reply debian/rules]: I'm not really an expert when looking at the code here but yeah that makes more sense to me at least.

**Pull request merged**